From cb7d95adf80a27ce1f48b0c167e9a4507563ff28 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sun, 12 Aug 2018 17:04:44 +0300
Subject: [PATCH] board: gaudi is a new ARM64 SoC Fixup

ONIX can't close timing on 50MHz, at 35MHz it does
---

diff --git a/board/habanalabs/gaudi/uboot_onix.fragment b/board/habanalabs/gaudi/uboot_onix.fragment
index 08d903e..bcf9e7a 100644
--- a/board/habanalabs/gaudi/uboot_onix.fragment
+++ b/board/habanalabs/gaudi/uboot_onix.fragment
@@ -1,8 +1,8 @@
-CONFIG_SYS_CLK_FREQ=50000000
+CONFIG_SYS_CLK_FREQ=35000000
 CONFIG_CONARGS="serial0:57600n8"
 CONFIG_BAUDRATE=57600
 CONFIG_BOOTDELAY=10
-CONFIG_DEBUG_UART_CLOCK=50000000
+CONFIG_DEBUG_UART_CLOCK=35000000
 CONFIG_BOOTCOMMAND="getfit && bootm $fileaddr#conf@1"
 CONFIG_DEFAULT_DEVICE_TREE="gaudi_onix"
 CONFIG_SPI_FLASH_STMICRO=y
